app/pods/publications/publication/documents/controller.js

```diff
@@ -25,18 +25,24 @@ export default class PublicationDocumentsController extends Controller {
   @tracked showTranslationModal = false;
   @tracked translateActivity = {
     mailContent: '',
+    mailSubject: '',
     finalTranslationDate: '',
     pieces: A([]),
   };
   @tracked previewActivity = {
     mailContent: '',
+    mailSubject: '',
     pieces: A([]),
   };
   @tracked selectedPieces = A([]);
 
   // Hacky way to refresh the checkboxes in the view without reloading the route.
   @tracked renderPieces = true;
 
+  concatNames(pieces) {
+    return pieces.map((piece) => piece.name).join('\n');
```

Comment:
met .name moeten we oppassen.
In de name kunnen we nu speciale characters steken zoals / \

Als we effectieve files gaan doorsturen gaan die een sanitized name hebben (staan zo reeds opgeslagen) zodat die downloadbaar en savebaar zijn.
Willen we dan die sanitized names tonen hier of de namen die je normaal in de applicatie ziet ?
De e-mail ontvanger gaat een file ontvangen met een licht andere tekst dan wat in de mail staat.

Je kan van piece de downloadFilename opvragen, dat is sanitized naam + extension.